subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishBasicIntegTest.groovy

```diff
@@ -273,7 +273,7 @@ class MavenPublishBasicIntegTest extends AbstractMavenPublishIntegTest {
         fails 'publish'
 
         then:
-        failure.assertHasCause("Maven publication 'maven' cannot include multiple components")
+        failure.assertHasCause("The value for property 'component' is final and cannot be changed any further.")
```

Comment:
❕ This seems like a worse error message.  Where does this property live?  There could be multiple publications and we wouldn't know which one.

Unless this context is also present elsewhere in the failure message?

Reply:
Reverted